Update creating-an-issue.md (#23536)
github/docs: content/issues/tracking-your-work-with-issues/creating-an-issue.md

`projects` | `https://github.com/octo-org/octo-repo/issues/new?title=Bug+fix&projects=octo-org/1` creates an issue with the title "Bug fix" and adds it to the organization's project board 1.
`template` | `https://github.com/octo-org/octo-repo/issues/new?template=issue_template.md` creates an issue with a template in the issue body. The `template` query parameter works with templates stored in an `ISSUE_TEMPLATE` subdirectory within the root, `docs/` or `.github/` directory in a repository. For more information, see "[Using templates to encourage useful issues and pull requests](/communities/using-templates-to-encourage-useful-issues-and-pull-requests)."

> {% tip %}
> 
> **Tip:** You can also target custom fields in your issue templates using URL query parameters.
> 
> {% endtip %}
> 

Reviewer: ```suggestion
{% tip %}

**Tip:** You can also target custom fields in your issue templates using URL query parameters.

{% endtip %}

```

See my comment below - there appears to already be information about this, but it is unclear. If we end up editing the below paragraph without a need for additional explanation, then we can remove this tip 👍

{% ifversion fpt or ghec %}

> To use query parameters to fill text fields by `id`, you can use issue forms. Query parameters for issue form fields can also be passed to the issue template chooser. For more information, see "[Syntax for {% data variables.product.company_short %}'s form schema](/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#keys)."

Reviewer: ```suggestion
You can also use URL query parameters to fill custom text fields that you have defined in issue form templates. Query parameters for issue form fields can also be passed to the issue template chooser. For more information, see "[Syntax for {% data variables.product.company_short %}'s form schema](/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#keys)."
```
I just looked into this and realized that the ability to target custom fields like this only applies to the YAML issue form templates ("[Syntax for issue forms](https://docs.github.com/en/enterprise-cloud@latest/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms)") which are only available for GitHub.com and Enterprise Cloud right now. I also think this above paragraph was trying to say that, but it was unclear and easy to miss. I'm suggesting just updating the first sentence of this paragraph to get this functionality across! Let me know if you think that is clear or if you still think it would be best to also put it in "Tip" form!

Author: ah yes, that makes sense.  I like it. I can remove the tip

`projects` | `https://github.com/octo-org/octo-repo/issues/new?title=Bug+fix&projects=octo-org/1` creates an issue with the title "Bug fix" and adds it to the organization's project board 1.
`template` | `https://github.com/octo-org/octo-repo/issues/new?template=issue_template.md` creates an issue with a template in the issue body. The `template` query parameter works with templates stored in an `ISSUE_TEMPLATE` subdirectory within the root, `docs/` or `.github/` directory in a repository. For more information, see "[Using templates to encourage useful issues and pull requests](/communities/using-templates-to-encourage-useful-issues-and-pull-requests)."

> {% tip %}
> 
> **Tip:** You can also target custom fields in your issue templates using URL query parameters.
> 
> {% endtip %}
> 

Reviewer: ```suggestion

```
Oops looks like the tip somehow didn't get removed, maybe my suggestion was formatted incorrectly! Going to suggest it again and go ahead and apply the change!
